src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/DeprecationCapability.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Linq;
+using NuGet.Versioning;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class DeprecationCapability : IDeprecated
```

Comment:
```suggestion
    public class DeprecationCapability : IDeprecation
```

I think naming the interface IDeprecation makes it seem more like the class implementing the interface has deprecation information and less like it's actually deprecated.